xc/common/utils/prjxray_create_ioplace.py

```diff
@@ -44,6 +54,12 @@ def main():
         default=sys.stdout,
         help='The output io.place file'
     )
+    parser.add_argument(
```

Comment:
I don't see any CMake change to actually pass `--synth_tiles` to create_ioplace.  Is this code even needed?